src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
 
-    p All #[a(href='#an-interactive-project') Interactive Projects] contain a default scene. Without any intervention from the #[a(href='#the-gameclient') Game Client] all participants and #[a(href='#groups') groups] will start on the default scene.
+    p All #[a(href='#an-interactive-project') Interactive Projects] has a default scene. Without any intervention from the #[a(href='#the-gameclient') Game Client] , all participants and #[a(href='#groups') groups] are shown the default scene.
 
-    p A #[a(href='#the-gameclient') Game Client] can change which scene a participant sees by updating the #[a(href='#groups') group] they belong to.
+    p The #[a(href='#the-gameclient') Game Client] can change the scene a participant sees by updating the #[a(href='#groups') group] they belong to.
 
     h3#groups Groups
-    p Groups are a way to group individual participants together. #[a(href='#participants') Participants] within a group all see the same #[a(href='#scenes') Scene] and can contribute input to the controls on that scene. #[a(href='#the-gameclient') Game Clients] can create and update groups at any time, including changing which scene the group is set to.
+    p Individual participants can be segmented into groups. Participants within a group all see the same Scene and contribute input to the controls that are part of that scene. #[a(href='#the-gameclient') Game Clients] can create and update groups any time, including changing the scene that the group is set to see.
 
-    p Groups can be used to create team-based experiences where groups compete to achieve a goal within the experience. For example, say a broadcaster is playing an adventure game. Interactive developers could create a "Helper" group and a "Hinderer" group. The help team would be able to heal the broadcaster and grant them buffs. The Hinderer team could spawn traps or monsters to try and get in the broadcasters way.
+    p Groups can be used to create team-based experiences where groups compete to achieve a goal within the experience. Using an adventure game again as an example, you could create a "Allies" group and a "Adversaries" group.
+    The ”Allies” group is provided with controls that gives them the ability to heal the character that the broadcaster is controlling or grant buffs to increase their chance of winning. Alternatively, the ”Adversaries” group could spawn traps or monsters to try to get in the way of winning - using their controls.
 
-    p All participants start out in the default group. #[a(href='#the-gameclient') Game Clients] can re-assign participants to any group from the #[a(href='#the-gameclient') Game Client].
+    p Participants always start out in the default group but #[a(href='#the-gameclient') Game Clients] can re-assign participants into any group.
```

Comment:
Participants always start out in the default group, however #[a(href='#the-gameclient') Game Clients] can re-assign participants into any group.